mergePDF/googless.py

```diff
@@ -0,0 +1,30 @@
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+import pandas as pd
+import numpy as np
+
+scope = ['https://spreadsheets.google.com/feeds', 'https://www.googleapis.com/auth/drive']
+creds = ServiceAccountCredentials.from_json_keyfile_name('client_secret.json', scope)
+# 
+
+client = gspread.authorize(creds)
+
+# Find a workbook by name and open the first sheet
+# Make sure you use the right name here.
+sheet = client.open("FLOCK Song List").sheet1
+
+# all_cells = sheet.get_all_records()
+# dates = sheet.row_values(2)
+
+date = sheet.find('1/5/2020')
```

Comment:
Date will be supplied by system when we kickoff the pdf assemble process for the week.